crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -29,8 +30,11 @@ pub(crate) struct BuildArgs {
 	#[arg(short = 'p', long)]
 	pub(crate) package: Option<String>,
 	/// For production, always build in release mode to exclude debug features.
-	#[clap(short, long)]
+	#[clap(short = 'r', long, conflicts_with = "profile")]
```

Comment:
You caa use only `short`, it will already use `r`:` #[clap(short, long, conflicts_with = "profile")]`

Reply:
Thank you!